pallets/ds-maps/src/lib.rs

```diff
@@ -68,20 +68,48 @@ impl<
         self.north_west.get_distance_vector(self.south_east)
     }
     
-    // Check tests for fn explanation
-    pub fn zone_intersects(self, target: Rect2D<Coord>) -> bool {
-        !(self.south_east.lon < target.north_west.lon || 
-          self.north_west.lon > target.south_east.lon ||
-          self.south_east.lat < target.north_west.lat || 
-          self.north_west.lat > target.south_east.lat)
+    /// True if this rect intersects other, excluding edges.
+    pub fn intersects_rect(self, target: Rect2D<Coord>) -> bool {
+        !(self.south_east.lon <= target.north_west.lon || 
+          self.north_west.lon >= target.south_east.lon ||
+          self.south_east.lat <= target.north_west.lat || 
+          self.north_west.lat >= target.south_east.lat)
+    }
+
+    /// True, if given point lies inside the rect, excluding edges. 
+    pub fn is_point_inside(&self, target: Point2D<Coord>) -> bool {
+        !(self.south_east.lon <= target.lon || 
+          self.north_west.lon >= target.lon ||
+          self.south_east.lat <= target.lat || 
+          self.north_west.lat >= target.lat)
+    }
+}
+
+#[cfg(test)]
+mod rect_tests {
+    use super::*;
+    use crate::tests::{construct_custom_rect, coord};
+    #[test]
+    fn rect_intersection_is_correct() {
+        let rect = construct_custom_rect("1", "1", "3", "3");
+        assert!(rect.intersects_rect(rect));
+        let partly_inside_rect = construct_custom_rect("0", "0", "2", "2");
+        assert!(rect.intersects_rect(partly_inside_rect));
+        let edge_rect = construct_custom_rect("0", "0", "1", "5");
+        assert!(!rect.intersects_rect(edge_rect));
+        let out_rect = construct_custom_rect("10", "10", "30", "30");
+        assert!(!rect.intersects_rect(out_rect));
     }
```

Comment:
Usually it is better to separate individual tests with an empty line for better readability.